replies.go

```diff
@@ -149,7 +154,7 @@ var replies = []Reply{
 	},
 	{
 		pattern: `(impact|install|use).*(wiki|spammer|multimc)`,
-		message: "Impact Wiki: https://github.com/ImpactDevelopment/ImpactIssues/wiki",
+		message: "For how to use Impact, please read the Wiki: https://github.com/ImpactDevelopment/ImpactIssues/wiki",
```

Comment:
The wiki covers more than just usage; installation and other FAQs are included there. Not sure what the wording should be here.